src/start.ts

```diff
@@ -235,21 +221,17 @@ export const handler = async (_event: any = {}): Promise<any> => {
               );
 
               // **************************************************************************
-              // 6. Make API request
+              // 5. Make API request
               // **************************************************************************
               node.logger.debug("making API request...", beaconIdLogOptions);
               const configOis = oises.find((o) => o.title === oisTitle)!;
               const configEndpoint = configOis.endpoints.find(
                 (e) => e.name === endpointName
               )!;
-              const configParameters = parameters.reduce(
-                (acc, p) => ({ ...acc, [p.name]: p.value }),
-                {}
-              );
-              const apiCallParameters = {
-                ...node.evm.encoding.safeDecode(template.parameters),
-                ...configParameters,
-              };
+              const apiCallParameters = [
+                ...parameters,
```

Comment:
This may be the wrong way around: you're overriding override parameters with template parameters?
We generally want configured parameters to override on-chain parameters.

Reply:
Good catch! I almost introduced the same bug again 😄